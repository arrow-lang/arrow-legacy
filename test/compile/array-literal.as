let explicit: int[3] = [1, 90, 20341];
let deduced = [51.12, 30, 1];
