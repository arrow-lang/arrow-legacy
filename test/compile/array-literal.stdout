@explicit = internal global [3 x i128] zeroinitializer
@deduced = internal global [3 x double] zeroinitializer

define internal void @array-literal..init() {
  store [3 x i128] [i128 1, i128 90, i128 20341], [3 x i128]* @explicit
  store [3 x double] [double 5.112000e+01, double 30, double 1], [3 x double]* @deduced
  ret void
}

define i32 @main(i32, i8**, i8**) {
  call void @array-literal..init()
  ret i32 0
}
